plugins/jobtype/jobtypes/hadoopShell/private.properties

```diff
@@ -0,0 +1,3 @@
+jobtype.class=azkaban.jobtype.HadoopShell
+
+command.blacklist.regex=(?i).*kinit.*
```

Comment:
Synced offline, it will be stated that comment in same line with execution code will not be allowed.